Tests/Brokerages/Tradier/TradierBrokerageHistoryProviderTests.cs

```diff
@@ -53,51 +60,23 @@ private static TestCaseData[] TestParameters
         public void GetsHistory(Symbol symbol, Resolution resolution, TimeSpan period, bool throwsException)
         {
             TestDelegate test = () =>
-            {
-                var useSandbox = Config.GetBool("tradier-use-sandbox");
-                var accountId = Config.Get("tradier-account-id");
-                var accessToken = Config.Get("tradier-access-token");
-
-                var brokerage = new TradierBrokerage(null, null, null, null, useSandbox, accountId, accessToken);
-
+            {   
                 var now = DateTime.UtcNow;
+                var request = new HistoryRequest(now.Add(-period),
+                    now,
+                    typeof(QuoteBar),
```

Comment:
`Tradier` only returns TradeBars really? think should probably return empty when requested quoteBars